docs/fsharp/whats-new/fsharp-9.md

```diff
@@ -0,0 +1,408 @@
+---
+title: What's new in F# 9 - F# Guide
+description: Find information on the new features available in F# 9.
+ms.date: 11/12/2024
+ms.topic: whats-new
+---
+
+# What's new in F# 9
+
+F# 9 introduces a range of enhancements that make your programs safer, more resilient, and performant. This article highlights the major changes in F# 9, developed in the [F#'s open source code repository](https://github.com/dotnet/fsharp).
+
+You can download the latest .NET SDK from the [.NET downloads page](https://dotnet.microsoft.com/download).
+
+## F# language changes
+
+This section describes updates to the language itself, changes you will notice when writing or reading F# code.
+
+### Nullable reference types
+
+Although F# is designed to avoid `null`, it can creep in when interfacing with .NET libraries written in C#. F# now provides a type-safe way to deal with reference types that can have `null` as a valid value.
+
+For more details, watch out for an [upcoming blog post about this feature](https://devblogs.microsoft.com/dotnet/tag/f/).
+
+Here are some examples:
+
+<pre><code class="language-fsharp">// Declared type at let-binding
```

Comment:
The class=language-fsharp is what is used in wordpress.
I guess in the dotnet/docs, you just use the regular triple-apostrophes with the "fsharp" moniker to make it rendere well.